client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import {
+  Button,
+  ButtonVariant,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  PageSection,
+  PageSectionVariants,
+  Text,
+  TextContent,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+} from "@patternfly/react-core";
+import { Table, Tbody, Th, Thead, Tr, Td } from "@patternfly/react-table";
+import { CubesIcon } from "@patternfly/react-icons";
+
+import { AppPlaceholder } from "@app/components/AppPlaceholder";
+import { ConditionalRender } from "@app/components/ConditionalRender";
+import { FilterToolbar, FilterType } from "@app/components/FilterToolbar";
+import { NotificationsContext } from "@app/components/NotificationsContext";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/components/TableControls";
+import { useLocalTableControls } from "@app/hooks/table-controls";
+import { useFetchArchetypes } from "@app/queries/archetypes";
+
+import ArchetypeApplicationsColumn from "./components/archetype-applications-column";
+import ArchetypeDescriptionColumn from "./components/archetype-description-column";
+import ArchetypeMaintainersColumn from "./components/archetype-maintainers-column";
+import ArchetypeTagsColumn from "./components/archetype-tags-column";
+
+const Archetypes: React.FC = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const { pushNotification } = React.useContext(NotificationsContext);
+
+  const { archetypes, isFetching, error: fetchError } = useFetchArchetypes();
+
+  const tableControls = useLocalTableControls({
+    idProperty: "id",
+    items: archetypes,
+    isLoading: isFetching,
+    isSelectable: false,
```

Comment:
If you want you can omit `isSelectable` and `expandableVariant` here because they will default to `false` and `null` respectively